x-pack/filebeat/docs/inputs/input-entity-analytics.asciidoc

```diff
@@ -0,0 +1,116 @@
+[role="xpack"]
+
+:type: entity-analytics
+
+[id="{beatname_lc}-input-{type}"]
+=== Entity Analytics Input
+
+++++
+<titleabbrev>Entity Analytics</titleabbrev>
+++++
+
+Experimental[Input]
```

Comment:
I am accustomed to see `experimental[]` to show a note. Is this the same thing?

Reply:
I was referencing the wrong file, then. There doesn't seem to be a consistent standard here:

```asciidoc
Beta[Input]
Beta [Input]
beta[]
experimental[]
```
🤷 

I'll use `experimental[]` since that's what `cel` is using and I like that format of that better.